src/styles/atoms/_atoms.link.scss

```diff
@@ -9,7 +9,7 @@
  * -------------------------------------------------------------------
  */
 
-a {
+a:not(.a-button) {
```

Comment:
Aan een knop en een link kan je beiden een icoontje toevoegen. Je moet dan voor beiden, naargelang de positie van het icoon, de klasse `has-icon-left` of `has-icon-right` toevoegen.

```
<button class="a-button has-icon-left">
    <i class="fa fa-arrow-left"></i>Normale knop
</button>

<a href="#" class="has-icon-left">
    <i class="fa fa-angle-left"></i>Dit is een mooie link
</a>
```

Het probleem ontstaat wanneer je een link hebt die op een knop moet lijken.

```
<a href="#" class="a-button has-icon-left">
    <i class="fa fa-arrow-left"></i>Normale knop
</a>
```

In dit geval gaat de het icoontje slecht uitgelijnd zijn omdat het hier de styling van `a.has-icon-left {}` toegepast krijgt, in plaats van de styling van `.a-button.has-icon-left {}`. Daarom heb ik in`atoms_link.scss` `a.has-icon-left {}` veranderd naar `a:not(.a-button).has-icon-left {}`. Hierdoor is dit probleem opgelost.